working/augmentation-libraries/feature-specification.md

```diff
@@ -705,10 +774,10 @@ It is a **compile-time error** if:
         variable, and that will not add any setter. An augmenting variable
         declaration only affects the initializer expression, not setters.
 
-*   A variable is augmented with another variable, and one is `late` and
-    the other is not. *(Augmentation cannot change `late`-ness, and since being
-    `late` does affect the initializer expression, the augmenting variable is
-    required to repeat the `late`.)*
+*   A variable is augmented by another variable declaration, and one is `late`
+    and the other is not. *(Augmentation cannot change `late`-ness, and since
+    being `late` does affect the initializer expression, the augmenting
+    variable is required to repeat the `late`.)*
```

Comment:
I could see expanding this later on, but I do think that being `late` is an important property to be aware of, and hiding that behind an augmentation seems wrong. So, I would rather not do it right now, even if it could be technically correct to allow it in one use case.

Reply:
ACK.